modules/statics/src/coins.ts

```diff
@@ -11885,6 +11885,46 @@ export const coins = CoinMap.fromCoins([
     '0x44ff8620b8ca30902395a7bd3f2407e1a091bf73',
     UnderlyingAsset['eth:virtual']
   ),
+  erc20(
```

Comment:
https://github.com/BitGo/BitGoJS/pull/5234 

shouldn't these be defined in the new file?